src/components/icons/loading.scss

```diff
@@ -18,12 +18,10 @@ $path2-gap: $section-length + $section-gap-2x;
   display: block;
   width: 30px;
   height: auto;
-  opacity: 0;
-  transition: all ease 0.3s;
-  pointer-events: none;
+  visibility: hidden;
 
   &--visible {
-    opacity: 1;
+    visibility: visible;
```

Comment:
This may trigger jank on chart animations because visibility can trigger layout, paint and composite where opacity is only paint, see https://csstriggers.com/visibility vs https://csstriggers.com/opacity.

I think it's best to revert this if possible?

Reply:
Will this be the case even if there's no transition happening here? But yeah sure.